packages/calcite-components/src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -774,9 +774,9 @@ export class InputDatePicker
 
   @State() private localeData: DateLocaleData;
 
-  private startInput: HTMLCalciteInputElement;
+  private startInput: HTMLCalciteInputTextElement;
```

Comment:
This was causing a type error in Lumina - the ref is assigned to calcite-input-text, but the type here was calcite-input.